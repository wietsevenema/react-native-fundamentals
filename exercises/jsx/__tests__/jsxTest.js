/* @flow */
import React from 'react';
import List from '../jsx';
import renderer from 'react-test-renderer';

describe('List', () => {
  it('renders ok', () => {
    const tree = renderer.create(
      <List />
    );
    expect(tree).toMatchSnapshot();
  });

});
