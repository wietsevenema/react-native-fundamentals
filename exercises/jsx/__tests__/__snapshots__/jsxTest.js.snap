// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List renders ok 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Apple
     
    2.99
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Orange
     
    3.25
  </Text>
</View>
`;
